dgraph/cmd/zero/run.go

```diff
@@ -83,7 +83,7 @@ instances to achieve high-availability.
 		"addr:port of this server, so other Dgraph alphas can talk to this.")
 	flag.IntP("port_offset", "o", 0,
 		"Value added to all listening port numbers. [Grpc=5080, HTTP=6080]")
-	flag.Uint64("idx", 1, "Unique node index for this server.")
+	flag.Uint64("idx", 1, "Unique node index for this server. idx cannot be 0.")
```

Comment:
@danielmai  Shall we update the docs here https://dgraph.io/docs/deploy/single-host-setup/#run-dgraph-zero ?